app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt

```diff
@@ -644,10 +647,38 @@ fun AppPresentationView(app: App) {
           overflow = TextOverflow.Ellipsis,
         )
       }
+      AppRating(rating = app.pRating)
     }
   }
 }
 
+@Composable
+fun AppRating(
+  modifier: Modifier = Modifier,
```

Comment:
Modifier is not necessary anymore. If kept than it should be applied on the parent row and not the children